README.md

```diff
@@ -35,31 +35,30 @@ The provided [napari][] plugin is a good example of how to stream for visualizat
 
 ### List devices
 
-```python
-import acquire
-print(acquire.Runtime().device_manager().devices())
+```python-repl
+>>> import acquire
+>>> print(acquire.Runtime().device_manager().devices())  # doctest: +SKIP
 ```
 
 ### Finite triggered acquisition
 
-```python
-import acquire
-runtime = acquire.Runtime()
-dm = runtime.device_manager()
-
-props = runtime.get_configuration()
-# select the first Hamamatsu camera
-props.video[0].camera.identifier = dm.select(DeviceKind.Camera, "hamamatsu.*")
-# stream to zarr
-props.video[0].storage.identifier = dm.select(DeviceKind.Storage, "zarr")
-props.video[0].storage.settings.filename = "out.zarr"
-props.video[0].camera.settings.shape = (2304, 2304)
-props.video[0].camera.settings.pixel_type = SampleType.U16
-props.video[0].max_frame_count = 100
-props = runtime.set_configuration(props)
-
-runtime.start()
-runtime.stop()  # wait for acquisition to complete
+```python-repl
+>>> import acquire
+>>> runtime = acquire.Runtime()
+>>> dm = runtime.device_manager()
+>>> props = runtime.get_configuration()
+>>> # select the first Hamamatsu camera
+>>> props.video[0].camera.identifier = dm.select(acquire.DeviceKind.Camera, "hamamatsu.*")
+>>> # stream to zarr
+>>> props.video[0].storage.identifier = dm.select(acquire.DeviceKind.Storage, "zarr")
+>>> props.video[0].storage.settings.filename = "out.zarr"
+>>> props.video[0].camera.settings.shape = (2304, 2304)
+>>> props.video[0].camera.settings.pixel_type = acquire.SampleType.U16
+>>> props.video[0].max_frame_count = 100
+>>> props = runtime.set_configuration(props)
+>>> runtime.start()  # doctest: +SKIP
+>>> # wait for acquisition to complete
+>>> runtime.stop()  # doctest: +SKIP
```

Comment:
@andy-sweet I kind of hate that doctest is forcing this to look like a REPL session. I'd appreciate any suggestions.

Reply:
While I like the `>>>` in Python source because it helps distinguish code from docs, agreed that I don't like it here in Markdown files.

I guess there are some ways to omit the `>>>` and extract the source as something to test and I even found [a pytest plugin that does this](https://github.com/modal-labs/pytest-markdown-docs).

But I'd lean towards just leaving the `>>>` in because certain Python repls (e.g. ipython) will just ignore them anyway.